src/measurement_kit/cmd/meek_fronted_requests.cpp

```diff
@@ -0,0 +1,53 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include "../cmdline.hpp"
+
+namespace meek_fronted_requests {
+
+#define USAGE                                                  \
+    "usage: measurement_kit [options] meek_fronted_requests\n" \
+    "                       [-B expected_body] input_file\n"
```

Comment:
For consistency with existing ooni-probe command line, there should be a `-f input_file` option rather than the possibility of receiving the input file after the options as an argument.